engine/internal/llmkind/llmkind.go

```diff
@@ -0,0 +1,12 @@
+package llmkind
```

Comment:
might want to move it to pkg, since `vector-store-manager` also will refer to ollama and vllm

Reply:
sounds good!